### PR TITLE
Remove app.modules.frontend_ui and update test set up

### PR DESCRIPTION
**Pull request by karenc**
We are no longer serving frontend using houston.  As long as
`docker-compose.yml` is used, we can access the frontend using nginx.

Frontend uses `POST /api/v1/auth/sessions` to login and `POST /logout`
to logout.

`/logout` was a route in `app/modules/frontend_ui.py` and is now
combined with `app.modules.app_ui.user_logout`.

`/logout` also needs to be added to the nginx config
`.dockerfiles/www/codex.conf` as a path to use houston.

Change integration-tests and testing github CI workflows to not build
frontend and just use dev-frontend.

### CODE REVIEW

**Review by hwindsor — APPROVED** [review]
I'm not claiming that I understand the significance of the changes bt if it means the tests pass ane the frontend_ui is removed, it's good by me

**Comment by hwindsor** [comment]
Actually, does this fix the 'integration tests not running locally' issue?
Does't that require the CODEX_URL in integration_tests/conftest.py to be corrected too?

File: .github/workflows/integration-tests.yml [hunk 1]
```diff
@@ -33,11 +35,15 @@ jobs:
...
-      - name: Build frontend and swagger ui
+      - name: Build swagger ui
         run: |
-          ./scripts/codex/build.frontend.sh
           ./scripts/swagger/build.sh
 
+      - name: Create apiKeys.js for frontend
+        run: |
+          cd _frontend.codex
+          cp src/constants/apiKeysTemplate.js src/constants/apiKeys.js
```
Comment (karenc) [inline .github/workflows/integration-tests.yml:45]: @mmulich `cp src/constants/apiKeysTemplate.js src/constants/apiKeys.js` is still necessary.  I'm wondering if it's better to include this in https://github.com/WildMeOrg/houston/blob/main/.dockerfiles/dev-frontend/docker-entrypoint.sh ?

Comment (colinwkingen) [reply .github/workflows/integration-tests.yml:45]: If we move this lets do it is a subsequent PR, I'd like to get this in right away so tests pass.

**Pull request merged**